docs/include/filter.asciidoc

```diff
@@ -118,8 +118,8 @@ endif::[]
   * Value type is {logstash-ref}/configuration-file-structure.html#boolean[boolean]
   * Default value is `true`
 
-Disable or enable metric logging for this specific plugin instance
-by default we record all the metrics we can, but you can disable metrics collection
+Disable or enable metric logging for this specific plugin instance.
+By default we record all the metrics we can, but you can disable metrics collection
 for a specific plugin.
```

Comment:
Thanks for making this correction, @jonbride.  This single-sourced file is used in all of our filter plugins, and so your change has made a big impact. Thank you for helping us improve the quality of our documentation.